MathlibTest/PPRoundtrip.lean

```diff
@@ -64,3 +65,21 @@ set_option linter.ppRoundtrip true in
 -- check that trailing comments do not trigger the linter
 example : 0 = 0 := by
   rw [] -- this goal is closed by the `rfl` implied by `rw`
+
+/--
+warning: Current syntax:  'le {a: Nat'
+Expected syntax: 'le {a : Na'
+
+note: this linter can be disabled with `set_option linter.style.commandStart false`
+-/
+#guard_msgs in
+example {a: Nat} : a = a := rfl
+
+/--
+warning: Current syntax:  ' {a :Nat} '
+Expected syntax: ' {a : Nat}'
+
+note: this linter can be disabled with `set_option linter.style.commandStart false`
+-/
+#guard_msgs in
+example {a :Nat} : a = a := rfl
```

Comment:
Does the linter also catch the following cases/can you add those as well, please?
```
example  {a :Nat} : a = a := rfl
example {a : Nat}{b : Nat} : a = a := rfl
example {a : Nat}  : a = a := rfl
example {alpha: Type*} [Neg alpha   ] {a : Nat} : a = a := rfl